server/app/schema.py

```diff
@@ -433,6 +433,12 @@ def resolve_custom_sets(self, info, **kwargs):
 
         return query.all()
 
+    custom_set_count = graphene.Int()
```

Comment:
Could we move this to be under `class NonNullConnection`, like this:

```
    total_count = graphene.Int()

    def resolve_total_count(self, info, **kwargs):
        return len(self.iterable)
```

Then you just add `totalCount` next to `edges` in your query